typescript/web/src/components/text-link.tsx

```diff
@@ -0,0 +1,14 @@
+import { Text } from "@chakra-ui/react";
+import NextLink from "next/link";
+
+export type TextLinkProps = {
+  href: string;
+  label: string;
+};
+export const TextLink = ({ href, label }: TextLinkProps) => (
```

Comment:
Thanks for the extraction effort 👍 

To answer your earlier question, we will start from your work to end-up having more high-level link component(s).  
It will prevent us from having to struggle with the specificities of `Link` and `NextLink` as we did on this PR and earlier. It will also let @geoffreyvancassel reuse it in /website so that he would be sure his links are always correct, regardless of their href or children.

As this component will quickly evolve until we have covered all cases, we'll have to make sure that it's OK at each change. In that sense, can you add a (very-simple) `text-link.stories.tsx` file too?

```tsx
import { Story } from "@storybook/react";
import { TextLink as TextLinkComponent } from "./text-link";
import { chakraDecorator, storybookTitle } from "../utils/stories";

export default {
  title: storybookTitle(TextLinkComponent),
  component: TextLinkComponent,
  decorators: [chakraDecorator],
};

export const TextLink: Story = () => <TextLinkComponent href="/">Back home</TextLinkComponent>;
```

(_A Jest test verifying could be nice too but it's less critical for now_)